README.md

```diff
@@ -41,20 +49,19 @@ your custom element project.
 ```html
 <!doctype html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
-  <script src="/bower_components/webcomponentsjs/webcomponents-loader.js"></script>
-  <link rel="import" href="/bower_components/iron-component-page/iron-component-page.html">
-</head>
-<body>
-  <iron-component-page></iron-component-page>
-</body>
+  <head>
+    <script type="module">
```

Comment:
Would it make sense to put a script to the polyfill based on this element's usage?

Reply:
/shrug